config/locales/en.yml

```diff
@@ -1148,6 +1148,9 @@ en:
   saving_credit_card: Saving credit card...
   card_has_been_removed: "Your card has been removed (number: %{number})"
   card_could_not_be_removed: Sorry, the card could not be removed
+  default?: Default?
+  authorised_shops: Authorised Shops
```

Comment:
It would be better to move these into their scope. It would look like `spree.users.cards.authorised_shops`, but I'm not 100% sure. And in the view: `t('.authorised_shops')`

Reply:
Done